pipelines/containerfiles/Containerfile.openvino.mlserver.mlflow

```diff
@@ -18,15 +18,8 @@ RUN mkdir /models && chown ovms:ovms /models
 # CHANGE THIS LINE TO MATCH YOUR MODEL
 COPY --chown=ovms:ovms $MODEL_DIR /models/1
 
-RUN <<EOF
-    # Not deleting this file causes a bug
-    if echo "${MODEL_DIR}" | grep -q "tf2model"; then rm -f /models/1/fingerprint.pb; fi
-
-    chmod o+rwX /models/1
-
-    # https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
-    chgrp -R 0 /models/1 && chmod -R g=u /models/1
-EOF
+                                # Not deleting this file causes a bug                                                # https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
```

Comment:
```suggestion
# Not deleting this file causes a bug
# https://docs.openshift.com/container-platform/4.13/openshift_images/create-images.html#use-uid_create-images
```

Reply:
@grdryn Here I opted for matching the comments to the actual part of that line, so it's easier to understand what belongs to which part. Or do you think it would be understandable having it at the start of the line?

Reply:
It's fine :+1: